public/app/features/integrated-alerting/components/Table/Table.styles.ts

```diff
@@ -0,0 +1,86 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
```

Comment:
If we use `useStyles` hook in the component,  then `stylesFactory` in not necessary, as advised here https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#usestyles-hook

Reply:
Thanks, missed to remove it. Done!